pkg/config/env/mongo.go

```diff
@@ -67,3 +72,52 @@ func MongoConnectionTimeoutWithDefault(value string) string {
 func MongoSSLWithDefault(value string) string {
 	return envy.Get("MONGO_SSL", value)
 }
+
+// MongoConnDetails returns mongo connection details defined by all the Mongo
+// environment variables
+func MongoConnDetails() (*conn.Details, error) {
+	var errs error
+	host, err := MongoHost()
+	if err != nil {
+		multierror.Append(errs, err)
+	}
+	portStr, err := MongoPort()
+	if err != nil {
+		multierror.Append(errs, err)
+	}
+	port, err := strconv.Atoi(portStr)
+	if err != nil {
+		multierror.Append(errs, err)
+	}
+	user, err := MongoUser()
+	if err != nil {
+		multierror.Append(errs, err)
+	}
+	pass, err := MongoPassword()
+	if err != nil {
+		multierror.Append(errs, err)
+	}
+	timeoutStr := MongoConnectionTimeoutWithDefault("1")
+	timeoutSec, err := strconv.Atoi(timeoutStr)
+	if err != nil {
+		multierror.Append(errs, errors.WithMessage(err, "invalid value for timeout"))
+	}
+	timeout := time.Duration(timeoutSec) * time.Second
+	sslStr := MongoSSLWithDefault("true")
+	ssl, err := strconv.ParseBool(sslStr)
+	if err != nil {
+		multierror.Append(errs, errors.WithMessage(err, "invalid value for mongo SSL"))
```

Comment:
I ❤️ the multierror approach, ain't nothing worse than resolve-deploy cycle 5 times learning missing value one at a time